client/my-sites/email/email-providers-comparison/index.jsx

```diff
@@ -486,20 +491,25 @@ class EmailProvidersComparison extends React.Component {
 		};
 
 		return (
-			<PromoCard
-				isPrimary
-				title={ translate( 'Get your own @%(domainName)s email address', translateArgs ) }
-				image={ image }
-				className="email-providers-comparison__action-panel"
-			>
-				<p>
-					{ translate(
-						'Pick one of our flexible options to connect your domain with email ' +
-							'and start getting emails @%(domainName)s today.',
-						translateArgs
-					) }
-				</p>
-			</PromoCard>
+			<>
+				<HeaderCake onClick={ this.goBack }>
+					{ translate( 'Emails' ) }
```

Comment:
A recent comment from Saygun made me realize that we should probably use `Add Email` instead of `Emails` in order to match the label of the button shown on the `Emails` page:

<img width="623" alt="screenshot" src="https://user-images.githubusercontent.com/594356/122886398-37a72280-d340-11eb-8183-3d9b386cc0fa.png">